kapre/time_frequency.py

```diff
@@ -617,68 +489,67 @@ class FreqAwareConv(Conv2D):
          (2018). An intriguing failing of convolutional neural networks and the coordconv
           solution. In Advances in Neural Information Processing Systems (pp. 9605-9616).
 
-    Arguments:
-        filters: Integer, the dimensionality of the output space (i.e. the number of
+    Args:
```

Comment:
nice, but just to confirm, i believe we're moving towards wrap all of them with `*args` and `**kwargs`?

Reply:
yup! I'll do this next. but we still have to keep them in the doc right?